src/Caching/Cache.php

```diff
@@ -98,6 +98,45 @@ public function load($key, $fallback = NULL)
 
 
 	/**
+	 * Reads multiple items from the cache
+	 * @param array
+	 * @param callable
+	 * @return array
+	 */
+	public function bulkLoad(array $keys, $fallback = NULL)
```

Comment:
So each time you call bulkLoad you will build an array of callbacks? that does not sound right.